<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>455</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>455</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QLabel" name="headerTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 22pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="SETable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>341</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="SEAddAttribute">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Code/Downloads/plus.png</normaloff>Code/Downloads/plus.png</iconset>
   </property>
  </widget>
  <widget class="QScrollArea" name="GBAttributeScroll">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>229</width>
      <height>239</height>
     </rect>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="SEOkButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SEExitButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Group by</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="SearchBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
